src/Idris/Main.idr

```diff
@@ -71,6 +71,7 @@ updateEnv
          defs <- get Ctxt
          addPkgDir "prelude"
          addPkgDir "base"
+         addPkgDir "contrib"
```

Comment:
I think `contrib` was deliberately excluded here - like in Idris1 you're supposed to load it manually.

Reply:
Thanks. I'll remove this.